vpc_peering/main.tf

```diff
@@ -0,0 +1,47 @@
+module "tags" {
+  source = "github.com/variant-inc/lazy-terraform//submodules/tags?ref=v1"
+
+  user_tags    = var.user_tags
+  name         = "tf-vpc-peering"
+  octopus_tags = var.octopus_tags
+}
+
+data "aws_vpc" "srcVpc" {
+  provider = aws.this
+  filter {
+    name   = "tag-value"
+    values = ["${var.name_tag_value}"]
+  }
+  filter {
+    name   = "tag-key"
+    values = ["Name"]
+  }
+}
+
+data "aws_vpc" "destVpc" {
+  provider = aws.peer
+  filter {
+    name   = "tag-value"
+    values = ["${var.name_tag_value}"]
+  }
+  filter {
+    name   = "tag-key"
+    values = ["Name"]
+  }
+}
```

Comment:
Why are you filtering src and destination VPCs with the same tag value? Would we not potentially need to filter src and dest with different names?